torchsynth/synth.py

```diff
@@ -238,30 +248,27 @@ def __init__(self, synthglobals: SynthGlobals, *args, **kwargs):
         # Register all modules as children
         self.add_synth_modules(
```

Comment:
We could clean up add_synth_modules so it plugs in synthglobals everywhere too.

So like you pass in:
```
["keyboard", MonophonicKeyboard]
["adsr_1", ADSR]
...
["mod_matrix", ModulationMixer, {"n_input": 4, "n_output": 5}]
```

But maybe the current way is cleanest and most explicit

But are you 100% sure that on_post_move_to_device is triggered in examples.py and the dumb non-lightning call:
```
voice = Voice().to(device)
```

I guess so? Since you can't create a voice on a device, you have to do `to`?

Reply:
Yup it is definitely triggered in examples.py, if it wasn't then it wouldn't work (we'd get different device complaints). I like your clean up idea though. I think that is great since then we don't need to pass in synthglobals for each one.